### DISCUSSION TITLE
What If Godot had its own Ad Network

### DISCUSSION THREAD

**Discussion by IbrahimSemab** [godotengine/godot-proposals#5924] [General Discussions]
I was wondering if godot had its own ad network. It was going to be great and super easy for all godot game developers. I recently made my google admob account to integrate ads to my game but I got struck while adding payments description, tax information and i couldn't do it well and my account got blocked and I think that I should suggest to you. I am sure that my suggestion would be accomplished.

**Comment by Calinou** [comment]
Godot is a game engine, not an advertising network :slightly_smiling_face: 
It's mainly developed by volunteers; an ad network is something that's best run by people who are dedicating their time to it full-time.

**Reply by IbrahimSemab** [reply]
But Unity is a game engine too that has its own ad network

**Reply by Calinou** [reply]
Unity happens to be owned by a company whose business model is maintaining an ad network. This is not the case of Godot.

**Reply by IbrahimSemab** [reply]
It means Godot is not running business Its just helping the game developers to show their skills.

**Reply by Zireael07** [reply]
Godot is explicitly NOT a business, it's a member of Software Freedom Conservancy which is a **non-profit** organization

**Comment by ghost** [comment]
> I was wondering if godot had its own ad network. It was going to be great and super easy for all godot game developers. I recently made my google admob account to integrate ads to my game but I got struck while adding payments description, tax information and i couldn't do it well and my account got blocked and I think that I should suggest to you. I am sure that my suggestion would be accomplished.

If you are facing trouble creating Google add account then you can knock me to get help.. i am going to help you for free. And you can try applovin Facebook.

**Reply by IbrahimSemab** [reply]
Hi hani09876! thanks in advance for your help but how can I contact you
